src/kirby/components/empty-state/empty-state.component.scss

```diff
@@ -0,0 +1,31 @@
+@import "../../scss/web-imports";
+
+article {
+  width: 330px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+}
+
+kirby-icon {
+  margin: auto;
+}
+
+h3,p {
+  margin: 0px;
+}
+
+kirby-icon + * {
+  margin-top: size('s');
+}
+
+h3 + .button-wrapper,
+p + .button-wrapper {
+  margin-top: size('s');
```

Comment:
Consider applying ` margin-bottom: size('s');` to `h3, p` instead unless you have a really good use case for doing the opposite :)

Reply:
Fixed